### DISCUSSION TITLE
Simple Digitaluhr oder generell Matrix ESP8266 12F

### DISCUSSION THREAD

**Discussion by MicGyver-72** [community/community#81439] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

Hallo Leute , da ich nicht weiss wie man das Programmieren kann und scheinbar auch kein anderer auf der Welt , vieleicht kann das jemand wie Du oder Ihr.
ich habe 2x 16x16 LED Neopixelmatten also 16x 32 Pixel und keine Einstellung in den ( teilweise unnützen da sich wiederholenden ) presets lässt mich eine simple Digitaluhr mit einem einfachen Countdown für Silvester einstellen ( ohne Laufschrift  )  oder einen stilisierten Weihnachtsschlitten über einer Linie die als Dorflandschaft durchgeht ( gelbe Wände rote Dächer ) oder mal ein Tannenbaum einstellen.
der "eingebaute" Countdown taugt nix , da ich den für 2024 auf 2077 einstellen muss  (total logich was ?).
kannst Du mir zufällig Bitte den Api code dafür erstellen , ansonsten ist der ganze Aufwand Mumpitz alles auf Englisch und da kann ich die Syntax nicht von der Ausgabe unterscheiden. Ich übersetze ja Programme und teilweise Spiele aber wenn ich über 7 GB runterladen muss über ein stupides Tool von Microsoft welches dann sein Kauderwelsch noch nicht mal im Internet findet um 970 KB zu Programmieren , dann hört bei mir das Verständniss auf. das Datum ist auch absolut nicht brauchbar da die Anzeige unlogisch ist weil Die Amerikaner gerne das Datum Rückwärts lesen . Umlaute gehen auch nicht ( also lieben die USAner ja wohl doch nicht die Deutschen so wie Propagiert )

also Meine Frage , kannst Du mir helfen?
Antwort auch auf Englisch , diesen Text Übersetze ich jetzt noch mal mit Google.

Mfg Mike aus Zwickau Sachsen Deutschland
..................................................................
Hello people, since I don't know how to program and apparently no one else in the world does, maybe someone like you can.
I have 2x 16x16 LED Neopixel mats i.e. 16x 32 pixels and no setting in the presets (sometimes useless because they are repetitive) allows me to set a simple digital clock with a simple countdown for New Year's Eve (without ticker text) or a stylized Christmas sleigh over a line that passes for a village landscape (yellow walls, red roofs) or put up a Christmas tree.
The “built-in” countdown is no use because I have to set it to 2077 for 2024 (totally logical, right?).
Can you please create the API code for me by chance, otherwise the whole effort is all in English and I can't distinguish the syntax from the output. I translate programs and sometimes games, but if I have to download over 7 GB using a stupid tool from Microsoft that can't even find its gibberish on the Internet in order to program 970 KB, then my understanding stops. The date is also absolutely unusable because the display is illogical because Americans like to read the date backwards. Umlauts don't work either (so the Americans probably don't love the Germans as much as they advertise)

So my question is, can you help me?
Answer also in English, I will now translate this text with Google.

Best regards Mike from Zwickau Saxony Germany

**Comment by Akash1134** [comment]
Hi @MicGyver-72 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
